net_unix.go

```diff
@@ -136,29 +136,29 @@ func (u *NetUNIX) parseLine(line string, hasInode bool, min int) (*NetUNIXLine,
 
 	users, err := u.parseUsers(fields[1])
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse ref count(%s): %v", fields[1], err)
+		return nil, fmt.Errorf("failed to parse ref count(%s): %w", fields[1], err)
```

Comment:
I guess this should be `"failed to parse ref count %q: %w"` in line with similar messages above.

Reply:
Yea, so much inconsistency her. :frowning_face: